homeassistant/components/pterodactyl/strings.json

```diff
@@ -7,8 +7,8 @@
           "api_key": "[%key:common::config_flow::data::api_key%]"
         },
         "data_description": {
-          "host": "The hostname or IP address of your Pterodactyl server (including the port).",
-          "api_key": "The client API key for accessing your Pterodactyl server."
+          "host": "The IP address or hostname of your Pterodactyl server, starting with either 'http://' or 'https://', optionally including the port at the end.",
```

Comment:
That's not the host, that's an URL. I would check using `YARL` to parse the input and get the pieces you need. Check how we do it in overseerr

Reply:
The library expects an URL:
`class PterodactylClient(
    url: Any | None = None,
    api_key: Any | None = None,
    backoff_factor: int = 1,
    retries: int = 3,
    extra_retry_codes: Any = [],
    debug: bool = False
)`

So I wouldn't need to "get the pieces". But I can replace `CONF_HOST` with `CONF_URL`.

Reply:
I replaced CONF_HOST with CONF_URL. I like it better. Please let me know if I have to do more here.

Reply:
I think it would still make sense to use YARL to parse anything to a standardized URL to give to the library